src/Interactive/CsiCore/project.json

```diff
@@ -1,6 +1,9 @@
 ﻿{
   "dependencies": { },
   "frameworks": {
+    "netstandard1.6": {
+      "imports": "portable-net452"
+    },
```

Comment:
It's working around #12458. There's a bug in the portable library tooling that is looking for something targeting netstandard to enable the proper compatibility checks. This is a hack to trick the project system into taking the right path, without actually affecting the build.